docs/dashboard/configuration/products.md

```diff
@@ -5,36 +5,53 @@ title: Product Types
+In this card you can determine if your products is physical and thus can be send to customers. To do that, select the [[Is&nbsp;this&nbsp;product&nbsp;shippable?]] checkbox. An additional field will then appear to note the weight of the products. Indicating the wieght is essential for calculating shipping costs correctly.
 
 ### Taxes
 
-Different goods are taxed differently across the world. If you leave the Tax Rate field blank, the system will assume that the product is classified as a standard taxable item. Click on the drop-down list to assign non-standard classifications. This is very important, as it means you are legally charging the correct amount for products. For example, educational materials may be tax exempt, while some hazardous substances have higher tax rates.
+Different goods are taxed differently across the world. If you leave the Tax Rate field blank, the system will assume that the product is classified as a standard taxable item. Click on the drop-down list to assign non-standard classifications. 
+
+This is very important, as it means you are legally charging the correct amount for products. For example, educational materials may be tax exempt, while some hazardous substances have higher tax rates.
+
 
+## How to add a product type
 
-### How to add a product type
+Click [[Add&nbsp;Product&nbsp;Type]] above the main product types list. Enter the required data into the Information, Shipping, and Taxes cards. 
 
-Click [[Add&nbsp;Product&nbsp;Type]] above the main product types list. The initial page has cards for Information, Shipping, and Taxes. A Product Attributes section also appears once the Product Type is saved.
+Once the new Product Type is saved, a Product Attributes card appears. 
 
 ![Adding product type](assets/dashboard-config/2.png)
 
 ### How to add attributes and variants
 
-Once you have created your Product Type and added tax and shipping information, click [[Save]] to create the new classification. An additional card will appear for adding Attributes and Variants to the Product Type. Add the new attribute or variant and then click [[Save]]. 
+Once you have created your Product Type and added tax and shipping information, click [[Save]]. An additional card will appear for adding Attributes and Variants to the Product Type. Click [[Assign&nbsp;Attribute]] and select the attributes from the list. 
+
+If your product has different variants, select the [[This&nbsp;product&nbsp;has&nbsp;variants]] checkbox to open a new Variant Attributes card. Click [[Assign&nbsp;Attribute]] and select the variants of your attribute. 
+ click [[Save]]. 
+
+> **Note**
+>
+> The lists of Attributes and Variant Attributes are interchangeable, and in fact they are one list. Therefore, when you are creating a Product Type, you are using one list when assigning new Product Attributes and Variant Attributes.
```

Comment:
"they are one list" - what do you mean by that?  Product Attributes and Variant Attributes are using the same "attribute" model underneath by they have a different purpose. Product Attributes are just simple properties of products while Variant Attributes determine what combinations of variants you can create. I'm not sure if this distinction is clear enough.